alembic/env.py

```diff
@@ -21,7 +21,11 @@
 import sys
 
 sys.path.append(os.getcwd())
-config.set_main_option('sqlalchemy.url', os.getenv('DEV_DATABASE_URL'))
+config.set_main_option(
+    'sqlalchemy.url',
+    os.getenv('TEST_DATABASE_URL') if os.getenv('APP_SETTINGS') == 'testing'
```

Comment:
Nice work @RonKbS I tested and introduced multiple heads intentionally and it worked. Looking at this setting, it caters for only settings and development, what will happen when the code is pushed to production. The environmental variable name for the Production database is `DATABASE_URL`. Can we have a look into this please.

Reply:
Thank you @michael-basweti. I consulted with @philophilo on this and he said that migrations are run differently on production and so there is no need to include a check for production settings here. As such the check here is for APP_SETTINGS=testing, otherwise it behaves as before.